pkg/kube/cluster-init.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/ash
+#
+# Copyright (c) 2023 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+K3S_VERSION=v1.26.3+k3s1
+KUBEVIRT_VERSION=v0.59.0
+LONGHORN_VERSION=v1.4.2
+CDI_VERSION=v1.56.0
```

Comment:
I'd like @deitch's thoughts on this. Clearly these are not part of the EVE image, but they are downloaded by default at bootup. Should we do anything from a SBoM perspective?

Reply:
This is a challenge. On the one hand, it really does change what software is installed. On the other hand, the SBoM normally reflects the software as distributed. We distribute eve-os as is, with an sbom (and even all sources). 

The fact that it _later_ downloads something else means that something else should have an SBoM if we care about it.

IMO there is nothing in the "bill of materials philosophy" that says, "ok, now you need to download beforehand everything you ever plan on downloading so you can create an SBoM for it; dynamic updates are banned."

I think for now we are ok, but we should reach out to the larger teams working on SBoMs and software supply chains to get a read on it.

Reply:
OK